back/model/Product.go

```diff
@@ -0,0 +1,19 @@
+package model
+
+import "gorm.io/gorm"
+
+type Product struct {
+	gorm.Model
+	Name                    string         `json:"name"`
+	Slug                    string         `json:"slug" gorm:"unique"`
+	CategoryID              uint           `json:"categoryID"`
+	Category                Category       `json:"category"`
+	PriceWithoutTaxes       int            `json:"PriceWithoutTaxes"`
+	PriceWithTaxes          int            `json:"priceWithTaxes"`
+	Description             string         `json:"description"`
+	ThumbnailUrl            string         `json:"thumbnailUrl"`
```

Comment:
Do you mean URI/ID or URL?

Reply:
It's the URL (we changed the typo later)

Reply:
🆗